app/views/pages/home.html.erb

```diff
@@ -1,8 +1,10 @@
 <div id="home">
+  <%= render "pages/partials/banner"%>
   <div class="container">
-    <%= render "pages/partials/membership" %>
+  <%= render "pages/partials/membership" %>
 
     <section id="planning">
+
       <h2>Planning</h2>
       <%= render "pages/partials/calendar" %>
     </section>
```

Comment:
The indentation isn't correct.
Can you also rmeove the empty line after the section.planning and add an empty line before the div.container please ?